### PR TITLE
Offline indicator and various fixes

### PR DESCRIPTION
**Pull request by MrTimscampi**
* Add a icon, informing the user that network connection is down
image
* Fix an issue where userInit would use an invalid token format ("token-ception", the valid token was contained in another token), leading to a blank screen when refreshing the app while logged in
* Add a server version check to the login flow in order to prepare for the new /users/me endpoint
* Make the scrollbars not look like garbage

### CODE REVIEW

**Comment by ferferga** [comment]
I think an snackbar message (with the failed progress bar on top maybe?) might be better for this? And, once user wants to switch between pages, we show the full page warning, which fades out once connection is established again.

Real life example that happens to me a lot: When I exit my flat, there's a 5 seconds or so period where my phone it's still holding to the WiFi connection, instead of switching to data. If I'm reading some movie details or scrolling through a page that has already everything loaded, it would be annoying to lock me out entirely because of this. Warning me with an snackbar about that is cool, so I can expect that things don't work as they should until the snackbar disappears!

Here doesn't happen that often but I now that many countries also have lots of trouble with data connections while commuting and such.

**Comment by ferferga** [comment]
Will review tomorrow, just wanted to drop some quick comments before going to bed.

Everything it's so much better now. However, I think that, alongside the icon, we should display an snackbar message for a few seconds while switching between connectivity states. Think of it as how Google Drive handles it — an snackbar-like message appears, warning you that you can still work but your work will only be saved once you get an internet connection. Then, you're left with an icon next to each of the documents that are available for offline storage.

Although the icon is visible and clear, a message will always catch the attention of the user, as the disconnection might have happened accidentally and you're not continuously focusing on it. The icon alone might get unnoticed in this situation.

File: components/LoginForm.vue [hunk 1]
```diff
@@ -73,27 +75,40 @@ export default Vue.extend({
     };
   },
   methods: {
+    ...mapActions('user', ['setUser', 'setUser']),
```
Comment (Camc314) [inline components/LoginForm.vue:78]:
```suggestion
    ...mapActions('user', ['setUser', 'clearUser']),
```
I might be wrong though

**Review by Camc314 — COMMENTED** [review]
Just one comment, rest looks good to me!

**Review by Camc314 — APPROVED** [review]

**Pull request merged**